src/core/Select/Select.test.tsx

```diff
@@ -202,14 +202,14 @@ it('should respect visible prop', () => {
   expect(select).toBeTruthy();
 
```

Comment:
Can you try changing our focus checks with this one `toHaveFocus()`?

Reply:
Changed in this file